### PR TITLE
Adjusted styles for orbit chat and added new color control with vars

### PR DESCRIPTION
**Pull request by bconfortin**
#8906

### CODE REVIEW

File: packages/augur-ui/src/modules/global-chat/components/global-chat.tsx [hunk 1]
```diff
@@ -48,7 +50,9 @@ export const GlobalChat = () => {
             <span>Global Chat</span>
             <button onClick={() => setShow(!show)}>{Close}</button>
           </div>
-          <iframe src="./chat/index.html#/channel/augur" />
+          {theme === THEMES.TRADING && <iframe src="./chat/index.html#/channel/augur" name={theme} />}
+          {theme === THEMES.BETTING && <iframe src="./chat/index.html#/channel/augur" name={theme} />}
+          {theme === THEMES.SPORTS && <iframe src="./chat/index.html#/channel/augur" name={theme} />}
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/global-chat/components/global-chat.tsx]: Tried to reload a few different ways but I really need this iframe to dismount and remount in order for the theme to kick in.

If you have any other solutions, please let me know.

Comment (JohnDanz) [reply packages/augur-ui/src/modules/global-chat/components/global-chat.tsx]: useMemo, have it re-render on theme. so `const chat = useMemo(() => renderIframe, [theme])` then `{chat}` in render

**Review by JohnDanz — CHANGES_REQUESTED** [review]

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**